odoo/addons/base/ir/ir_model.py

```diff
@@ -465,6 +465,13 @@ def _custom_many2many_names(self, model_name, comodel_name):
     def _onchange_ttype(self):
         self.copy = (self.ttype != 'one2many')
         if self.ttype == 'many2many' and self.model_id and self.relation:
+            if not self.env.get(self.relation):
+                return {
+                    'warning': {
+                        'title': _('Model %s does not exist' % self.relation),
```

Comment:
do not put the variable inside the translation clause
https://www.odoo.com/documentation/11.0/reference/translations.html#variables
(also could be super cool if you could reflect the changes in the .pot file too)

Reply:
Good catch! I updated the whole pot file for the base module, not sure if you usually handle it like that ?

Reply:
haaaa why do I see this just 5 seconds after trying to push two more commits ?

Reply:
@tgroutars no worries, i handle this. I'll do another commit on top of the one i already merged.